src/common/feature-flags.ts

```diff
@@ -100,10 +100,10 @@ export function getAllFeatureFlagDetails(): FeatureFlagDetail[] {
         },
         {
             id: FeatureFlags.tabStopsAutomation,
-            defaultValue: false,
+            defaultValue: true,
             displayableName: 'Tab Stops Automation',
             displayableDescription: 'Enables the new tab stops automation',
-            isPreviewFeature: false,
+            isPreviewFeature: true,
             forceDefault: false,
```

Comment:
```suggestion
            forceDefault: true,
```

We should forceDefault as well, since we want everyone to be using it.